vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleSpreadsheetLayout.java

```diff
@@ -106,26 +116,60 @@ private boolean isInputValid() {
   private static class SampleRegistrationSheetBuilder {
 
     private final SampleRegistrationService sampleRegistrationService;
+    private static Experiment activeExperiment;
+
+    private static int numberOfSamples;
 
     public SampleRegistrationSheetBuilder(SampleRegistrationService sampleRegistrationService) {
       this.sampleRegistrationService = sampleRegistrationService;
     }
 
+    public static void setActiveExperiment(Experiment experiment) {
+      activeExperiment = experiment;
+      List<ExperimentalGroup> groups = activeExperiment.getExperimentalGroups().stream().toList();
+      numberOfSamples = 3*groups.stream().map(ExperimentalGroup::sampleSize)
+          .mapToInt(Integer::intValue).sum();
```

Comment:
As discussed today it'd make more sense to show as much rows as fit on the screen size with the option to add an additional row with the same styling 👍